arangosh/Benchmark/BenchFeature.cpp

```diff
@@ -287,6 +335,9 @@ void BenchFeature::start() {
         requestTime,
     });
     for (size_t i = 0; i < static_cast<size_t>(_concurrency); ++i) {
+      if (_duration != 0) {
+        _realOperations = threads[i]->_counter;
```

Comment:
Assigning to the same value in a loop...
Does this need to be a `+=`, by the way?
The intent of the code is not clear to me, at least.

Reply:
yes, definitely!

Reply:
fixed.